renderer/src/components/FilAddressForm.tsx

```diff
@@ -25,6 +25,10 @@ const checkAddressString = (address: string) => {
   } else {
     throw new Error('Invalid address type')
   }
+  const res = await fetch(`https://station-wallet-screening.fly.dev/${address}`)
```

Comment:
Have you considered moving this check to the `main` part?

I see the following benefits:
- We don't need the screening service to support CORS
- It may be easier to write tests for this check

Reply:
I thought about it, yes!

> * We don't need the screening service to support CORS

Adding CORS support to the screening service is a lot less logic than adding another Desktop IPC call unfortunately. Plus, debugging is easier when you see the HTTP request in the network tab.

> * It may be easier to write tests for this check

I propose to reevaluate that when we are going to write the test

Reply:
Fair enough 👍🏻